core/src/main/scala/cats/data/IndexedStateT.scala

```diff
@@ -247,6 +246,23 @@ private[data] sealed abstract class IndexedStateTInstances extends IndexedStateT
   implicit def catsDataAlternativeForIndexedStateT[F[_], S](implicit FM: Monad[F],
     FA: Alternative[F]): Alternative[IndexedStateT[F, S, S, ?]] with Monad[IndexedStateT[F, S, S, ?]] =
     new IndexedStateTAlternative[F, S] { implicit def F = FM; implicit def G = FA }
+
+  implicit def catsErrorControlForStateT[F[_], G[_], S, E]
+  (implicit E: ErrorControl[F, G, E], M: Monad[G]): ErrorControl[StateT[F, S, ?], StateT[G, S, ?], E] =
```

Comment:
`M: Monad[G]` no longer required since it's provided by `ErrorControl` right?

Reply:
Yep, will fix, this was needed when G was an Applicative so that `StateT[G, S, A]` could be an Applicative